.github/release.yml

```
@@ -0,0 +1,16 @@
+# .github/release.yml
+
+changelog:
```

Is this header necessary?

```suggestion
changelog:
```